### PR TITLE
Edits in PR 21730

### PR DESCRIPTION
**Pull request by JarbasHorst**
Fixed issues in PR #21730.

### CODE REVIEW

File: api-reference/v1.0/resources/security-kubernetesserviceport.md [hunk 1]
```diff
@@ -11,30 +11,29 @@ doc_type: resourcePageType
...
 ### containerPortProtocol values
 
 | Member             | Description                                       |
 |:-------------------|:--------------------------------------------------|
-| udp                | User Datagram Protocol.                           |
-| tcp                | Transmission Control Protocol.                    |
-| sctp               | Stream Control Transmission Protocol.             |
+| UDP                | User Datagram Protocol.                           |
+| TCP                | Transmission Control Protocol.                    |
+| SCTP               | Stream Control Transmission Protocol.             |
```
Comment (MSRonBorysowski) [inline api-reference/v1.0/resources/security-kubernetesserviceport.md:34]: this should be lower cased

File: api-reference/beta/resources/dictionary.md [hunk 2]
```diff
@@ -9,28 +9,33 @@ doc_type: resourcePageType
 
 # dictionary resource type
 
-Namespace: microsoft.graph.security
```
Comment (MSRonBorysowski) [inline api-reference/beta/resources/dictionary.md:12]: why removing the security namespace? this type was created for the security namespace

File: api-reference/beta/resources/security-filehash.md [hunk 3]
```diff
@@ -13,25 +13,26 @@ Namespace: microsoft.graph.security
...
 ## Properties
 
-| Property   | Type                                                                    | Description                             |
-|:-----------|:------------------------------------------------------------------------|:----------------------------------------|
+| Property   | Type                                                                    | Description                                                           |
+|:-----------|:------------------------------------------------------------------------|:----------------------------------------------------------------------|
+| algorithm  | [microsoft.graph.security.fileHashAlgorithm](#filehashalgorithm-values) | The hash algorithm type. Possible values are: `unknown`, `md5`, `sha1`, `sha256`, `sha256ac`, `unknownFutureValue`. |
 | value      | String                                                                  | The hash value. |
-| algorithm  | [microsoft.graph.security.fileHashAlgorithm](#filehashalgorithm-values) | The hash algorithm type.                |
+
```
Comment (MSRonBorysowski) [inline api-reference/beta/resources/security-filehash.md:24]: why removing this property?

File: api-reference/beta/resources/security-kubernetescontrollerevidence.md [hunk 4]
```diff
@@ -24,10 +24,10 @@ Inherits from [alertEvidence](../resources/security-alertevidence.md).
 |createdDateTime|DateTimeOffset|The date and time when the evidence was created and added to the alert. The Timestamp type represents date and time information using ISO 8601 format and is always in UTC time. For example, midnight UTC on Jan 1, 2014 is `2014-01-01T00:00:00Z`. Inherited from [alertEvidence](../resources/security-alertevidence.md). |
 |name|String|The controller name.|
 |namespace|[microsoft.graph.security.kubernetesNamespaceEvidence](./security-kubernetesnamespaceevidence.md)|The service account namespace.|
-|labels|[microsoft.graph.security.dictionary](./security-dictionary.md)|The labels for the Kubernetes pod.|
+|labels|[microsoft.graph.dictionary](./dictionary.md)|The labels for the Kubernetes pod.|
```
Comment (MSRonBorysowski) [inline api-reference/beta/resources/security-kubernetescontrollerevidence.md:27]: should be in security namespace

File: api-reference/beta/resources/security-kubernetesserviceport.md [hunk 5]
```diff
@@ -13,27 +13,26 @@ Namespace: microsoft.graph.security
...
 ### containerPortProtocol values
 
 | Member             | Description                                       |
 |:-------------------|:--------------------------------------------------|
-| udp                | User Datagram Protocol.                           |
-| tcp                | Transmission Control Protocol.                    |
-| sctp               | Stream Control Transmission Protocol.             |
+| UDP                | User Datagram Protocol.                           |
+| TCP                | Transmission Control Protocol.                    |
+| SCTP               | Stream Control Transmission Protocol.             |
```
Comment (MSRonBorysowski) [inline api-reference/beta/resources/security-kubernetesserviceport.md:35]: should be lowercased

File: api-reference/v1.0/resources/dictionary.md [hunk 6]
```diff
@@ -9,26 +9,31 @@ doc_type: resourcePageType
 
 # dictionary resource type
 
-Namespace: microsoft.graph.security
+Namespace: microsoft.graph
```
Comment (MSRonBorysowski) [inline api-reference/v1.0/resources/dictionary.md:12]: security namespace

File: api-reference/beta/resources/security-filehash.md (hunk 3 above)
Comment (JarbasHorst) [reply api-reference/beta/resources/security-filehash.md:24]: Not removed, just sorted in alphabetical order.

**Pull request merged**